src/core/Tooltip/Tooltip.tsx

```diff
@@ -33,17 +33,8 @@ export type TooltipProps = {
  * <Button ref={buttonRef} />
  * <Tooltip content='tooltip text' reference={buttonRef} />
  */
-export const Tooltip = (props: TooltipProps) => {
-  const {
-    content,
-    children,
-    className,
-    style,
-    visible,
-    ref,
-    id,
-    ...rest
-  } = props;
+export const Tooltip = React.forwardRef<Element, TooltipProps>((props, ref) => {
+  const { content, children, className, style, visible, id, ...rest } = props;
```

Comment:
So even though we pass ref on the `Popover` component, at the end it appears on children? (according to your test)
Then I am lost a little, whats the point of this ref if user can add ref on his provided children himself...

Reply:
I was a little surprised too, that it appears on children.

I think the point is to allow nesting tippys. See linked issue and also see #296

Reply:
So now the scenario mentioned is working well? without the need of div?

Reply:
Yes, I tested this
![image](https://user-images.githubusercontent.com/9084735/131504106-e4f45124-cba5-4607-8783-3c7a268c68ca.png)
![image](https://user-images.githubusercontent.com/9084735/131504120-7dec6239-5575-450e-aa51-3e45e46458fc.png)